package.json

```diff
@@ -125,15 +128,15 @@
   "husky": {
     "hooks": {
       "pre-commit": "npm run test && npm run lint",
-      "pre-push": "commitlint --verbose --from=origin/master"
+      "pre-push": "commitlint --verbose --from=origin/main"
     }
   },
   "scripts": {
     "changelog": "standard-changelog -i CHANGELOG.md -s -r 1",
     "build": "webpack --mode production --env production",
     "dev": "webpack --progress --colors",
     "dist": "npm run lint && npm run test && npm run build && npm run docs",
-    "docs": "typedoc --out dist/docs src",
+    "docs": "typedoc --out dist/docs/ src && antora generate local-site.yml --to-dir docs/",
```

Comment:
Just FYI: Not sure if this is helpful. I use `antora --stacktrace generate` it gives a little bit more output when we generate the docs in case of an error.